frontend/src/metabase/components/ListItem/ListItem.jsx

```diff
@@ -1,47 +1,59 @@
 /* eslint "react/prop-types": "warn" */
-import { memo } from "react";
+import cx from "classnames";
 import PropTypes from "prop-types";
+import { memo } from "react";
 import { Link } from "react-router";
-import cx from "classnames";
-import Ellipsified from "metabase/core/components/Ellipsified";
+
 import Card from "metabase/components/Card";
 import S from "metabase/components/List/List.css";
+import Ellipsified from "metabase/core/components/Ellipsified";
 import { Icon } from "metabase/core/components/Icon";
 
-const ListItem = ({ name, description, placeholder, url, icon }) => (
-  <li className="relative">
-    <Link to={url} className="text-brand-hover">
-      <Card hoverable className="mb2 p3 bg-white rounded bordered">
-        <div className={cx(S.item)}>
-          <div className={S.itemIcons}>
-            {icon && <Icon className={S.chartIcon} name={icon} size={16} />}
+import { Root } from "./ListItem.styled";
+
+const ListItem = ({ name, description, disabled, placeholder, url, icon }) => {
+  const card = (
```

Comment:
`card` is an existing code, it's just extracted to a variable so it can be reused in conditional `return` statements.